.changeset/quick-elephants-yawn.md

```diff
@@ -0,0 +1,5 @@
+---
+"@khanacademy/wb-dev-build-settings": major
```

Comment:
Since most of the package's `package.json` dependencies were updated to remove `@babel/runtime`, do we need a changeset for those packages so a new version of it is released? 

I think all the packages will get patch updated anyways since they use `wb-dev-build-settings` and it's a major change, but wanted to check!

Reply:
That's how I thought about it :). All packages should get `patch` updates from the `major` change.